doc/adr/2022-05-06-customers-must-type-experimental-to-access-experimental-features.md

```diff
@@ -0,0 +1,53 @@
+**Title**: customers must type `[Ee]xperimental` to access experimental features.
```

Comment:
what about a `GOOGLE_CLOUD_CPP_EXPERIMENTAL_FOO=...` environment variable?

Reply:
Fixed in the "decision" section